### PR TITLE
test_s3_engine_heavy_write_check_mem: turn test off

### PR DESCRIPTION
**Pull request by CheSema**
It is pointless to fix that test now.
There is a significant impact from ParallelFormattingOutputFormat and its settings.
Firstly that task has to be done https://github.com/ClickHouse/ClickHouse/issues/51152

In contrast the similar test for s3 disk test_s3_disk_heavy_write_check_mem is pretty stable. S3 disk does not use ParallelFormattingOutputFormat. 

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Turn off flaky test test_s3_engine_heavy_write_check_mem until the memory consumption is stable.

### CODE REVIEW

**Comment by robot-clickhouse-ci-2** [comment]
*This is an automated comment for commit 524edd8460431b447c4bcd2f932bed489f7a83f6 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/57025/524edd8460431b447c4bcd2f932bed489f7a83f6/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6936928954">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/57025/524edd8460431b447c4bcd2f932bed489f7a83f6/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: tests/integration/test_merge_tree_s3/test.py [hunk 1]
```diff
@@ -870,12 +874,18 @@ def test_s3_engine_heavy_write_check_mem(
...
+        10 * 1024 * 1024, timeout=slow_timeout, count=slow_responces
+    )
 
     query_id = f"INSERT_INTO_S3_ENGINE_QUERY_ID_{in_flight}"
     node.query(
         "INSERT INTO s3_test SELECT number, toString(number) FROM numbers(50000000)"
-        f" SETTINGS max_memory_usage={2*memory}"
+        f" SETTINGS "
+        f" max_memory_usage={2*memory}"
+        # f", max_threads=1" # ParallelFormattingOutputFormat consumption depends on it
```
[the remark this answers is not in the document]
Comment (CheSema) [reply tests/integration/test_merge_tree_s3/test.py:888]: This is a hint (the comment inside the comment) what to do with that test in scope of ParallelFormattingOutputFormat.
I can uncomment that line the test is going to be turned off any way.

[the remark this answers is not in the document]
Comment (CheSema) [reply tests/integration/test_merge_tree_s3/test.py:888]:
```suggestion
        f", max_threads=1" # ParallelFormattingOutputFormat consumption depends on it
```

**Review by SmitaRKulkarni — APPROVED** [review]
LGTM

File: tests/integration/test_merge_tree_s3/test.py [hunk 2]
```diff
@@ -870,12 +874,18 @@ def test_s3_engine_heavy_write_check_mem(
...
+        10 * 1024 * 1024, timeout=slow_timeout, count=slow_responces
+    )
 
     query_id = f"INSERT_INTO_S3_ENGINE_QUERY_ID_{in_flight}"
     node.query(
         "INSERT INTO s3_test SELECT number, toString(number) FROM numbers(50000000)"
-        f" SETTINGS max_memory_usage={2*memory}"
+        f" SETTINGS "
+        f" max_memory_usage={2*memory}"
+        f", max_threads=1" # ParallelFormattingOutputFormat consumption depends on it
```
Comment (CheSema) [inline tests/integration/test_merge_tree_s3/test.py:888]:
```suggestion
        f", max_threads=1"  # ParallelFormattingOutputFormat consumption depends on it
```

**Pull request merged**